.travis.yml

```diff
@@ -1,5 +1,18 @@
-sudo: false
```

Comment:
Yes, I removed the sudo command as @raffenet mentioned and added back this line.